### ISSUE TITLE
[A11Y] OJS/OPS Reader View - "About" menu using non-native HTML element (button)

### ISSUE THREAD

**Issue by israelcefrin** [pkp/pkp-lib#12654]
## Issue Description
The "About" section in the navigation is currently a static link rather than a button, and there is no direct navigation link back to the main site, which hinders consistent navigation.

## How to reproduce / Current behavior
1. View the OJS/OPS top navigation.
2. Click "About" and note it is not a dropdown, it is a link .
3. Observe the lack of a "Home" or "Main Page" link in the menu.

## What application are you using?
OJS/OPS version 3.5.0

## Stack used 
- Device: Desktop
- OS: Windows/macOS
- Browser: Chrome browser

## Testing method
- Method: Manual test

## Action
1. Convert the "About" link into a dropdown menu using `button` native element.

### Screenshoot

<img width="883" height="97" alt="About link in top navigation" src="https://github.com/user-attachments/assets/e850444b-8d3c-4c00-8234-5ca213a6f471" />

### WCAG Success Criteria
3.2.3 Consistent Navigation (AA)
